### DISCUSSION TITLE
What's the default renderer of niri

### DISCUSSION THREAD

**Discussion by sabisho** [YaLTeR/niri#220] [Q&A]
Which rendering api is used in niri opengl or vulkan.
I really hope it defaults to vulkan as it is more efficient fast and very good with all the 10-bit colors and HDR stuff.

**Comment by YaLTeR** [comment]
OpenGL, there's only OpenGL renderer in Smithay. For compositors Vulkan efficiency doesn't matter much because compositors only do extremely simple drawing. There's no HDR support on Wayland in general at the moment, but work is ongoing.

**Comment by sabisho** [comment]
Then will the games that run using vulkan run on niri for example CS2 ?
Sorry but I don't know about graphics stuff.

**Reply by YaLTeR** [reply]
Yeah of course.

**Reply by sabisho** [reply]
And they will use Vulkan right ? BTW thanks for the quick replies.

**Reply by YaLTeR** [reply]
Yes. The apps can use whatever means they want to render.

**Comment by sabisho** [comment]
Thanks!
